clang/lib/Sema/SemaChecking.cpp

```diff
@@ -2639,10 +2651,16 @@ Sema::CheckBuiltinFunctionCall(FunctionDecl *FDecl, unsigned BuiltinID,
     }
     break;
   }
+  case Builtin::BI__builtin_cheri_cap_load_tags:
```

Comment:
I guess we could expand this to lc+cgettag, but I think an error if it's not support might make it clearer that the optimized version does not exist?